src/components/request/confirmRequest.js

```diff
@@ -21,6 +21,7 @@ class ConfirmRequest extends React.Component {
     } = this.props;
     let link = `lisk://wallet?recipient=${address}&amount=${amount}`;
     link = reference ? `${link}&reference=${reference}` : link;
+    link = encodeURI(link);
```

Comment:
I would rather use `reference=${encodeURIComponent(reference)}` that way it also encodes/allows chars like `&`